src/components/Admin/custom/PostPreview.tsx

```diff
@@ -0,0 +1,47 @@
+import {Stack} from '@mui/material'
+import Grid from '@mui/material/Unstable_Grid2'
+import {FC, useState} from 'react'
+import {FormDataConsumer, ThemeProvider} from 'react-admin'
+
+import {Post} from '@/components/Posts/Post'
+import {PostDetail} from '@/components/Posts/PostDetail'
+import {theme} from '@/theme'
+
+export const PostPreview: FC = () => {
+  const [isDetailOpen, openDetail] = useState<boolean>()
+  return (
+    <ThemeProvider theme={theme}>
+      <FormDataConsumer>
+        {({formData}) => {
+          return (
+            <>
+              <Grid container>
+                <Grid xs={6}>
+                  <Stack gap={5}>
+                    <Post
```

Comment:
aasi ok... vravim si, ked pozeram na ten Posts.tsx, ze by sa dalo extrahovat viac, cele UI postov, a v arrayi posts by si tam tuto poslal jeden element. ale asi to teraz nestoji za dalsi effort